packages/mint-components/src/components/sqm-text/sqm-text.tsx

```diff
@@ -1,54 +1,54 @@
 import { Component, h, State } from "@stencil/core";
 import { withHooks } from "@saasquatch/stencil-hooks";
-import { insertCSS } from "../../insertcss";
-import debugFn from "debug";
+// import { insertCSS } from "../../insertcss";
```

Comment:
Should be deleted instead of commented out

Reply:
done!